observable.go

```diff
@@ -264,15 +265,23 @@ func (o *observable) Take(nth uint) Observable {
 func (o *observable) TakeLast(nth uint) Observable {
 	out := make(chan interface{})
 	go func() {
-		buf := make([]interface{}, nth)
+		n := int(nth)
```

Comment:
Since a recent merge, there's a new way to implement operators. Hence, it triggered a conflict here.